docs/surveys/2020-jupyterlab-survey.md

```diff
@@ -0,0 +1,363 @@
+    - Usage patterns.
+    - Data
+    - Visualization.
+    - Scale.
+    - Collaboration.
+
+
+### Usage Patterns
+
+1. How frequently do you use Jupyter?
+    - Daily: 4 or more hours per day.
+    - Daily: 3 or less hours per day. 
+    - Weekly.
+    - Monthly.
+    - I no longer use Jupyter.
+    - I have never used Jupyter.
+
+2. How long have you been using Jupyter?
+    - 2+ years.
+    - 1-2 years.
+    - 6-12 months.
+    - Less than 6 months (welcome =]).
+    - I don't use Jupyter.
+
+3. What languages do you use in Jupyter? (pick up to 4)
+    - C (and derivatives)
+    - Groovy
+    - Java
+    - JavaScript
+    - NodeJS
+    - Perl
+    - PHP
+    - Python
+    - R
+    - Ruby
+    - Rust
+    - Scala
+    - Spark SQL
+    - SQL
+    - TypeScript
+    - ❗I wrap/ use bindings for other languages.
+    - ❗My preferred language is not supported in Jupyter.
+    - Other (please specify)
+
+4. What are your primary job roles when you are using Jupyter? (pick up to 2)
+    - Backend engineer.
+    - Business analyst.
+    - Data engineer.
+    - Data scientist.
+    - Database Admin (DBA).
+    - DevOps.
+    - Financial modeler/ analyst.
+    - Front end/ web development.
+    - Infrastructure engineer/ cloud architect.
+    - Scientist/ researcher.
+    - Student.
+    - Sysadmin.
+    - Teacher/ lecturer.
+    - Tutor/ teaching assistant.
+    - Other (please specify)
+
+5. What are your go-to tools for performing data science, scientific computing, 
+and machine learning on your laptop/ desktop (non-cloud) for data science? (pick up to 3)
+    - Spyder.
+    - PyCharm.
+    - nteract.
+    - Atom.
+    - Emacs.
+    - Zeppelin.
+    - iPython.
+    - VS Code.
+    - JupyterLab
+    - RStudio.
+    - Jupyter Notebook (classic)
+    - Vim.
+    - Sublime Text.
+    - Other (please specify)
+
+6. How do you run and/ or access Jupyter? (pick up to 4)
+    - Run directly on local machine (laptop, desktop).
+    - Through a Python virtual environment (e.g. conda, venv)
+    - Through Docker.
+    - HPC or on-premise server.
+    - Cloud server (e.g. EC2).
+    - JupyterHub.
+    - BinderHub / MyBinder.
+    - Cloud service - AWS: EMR, SageMaker.
+    - Cloud service - Azure: Notebooks, ML Studio.
+    - Cloud service - Databricks
+    - Cloud service - Google: AI Platform, Dataproc.
+    - Cloud service - IBM: Watson.
+    - Google Colab.
+    - Through a 3rd party SaaS app (e.g. Observable, CoCalc)
+    - ❓Don’t know how, I just go to a URL.
+    - Other (please specify)
+
+7. What tasks do you need to perform and what tools do you use to accomplish them?
+    - Writing a software package.
+    - Visualize data in charts, plots, or dashboards.
+    - Writing software documentation.
+    - Building a machine learning or statistical model.
+    - Cleaning and preparing data.
+    - Run pipelines, workflows, or ETL jobs.
```

Comment:
Maybe expand on what ETL means (just to be as clear as we can?)

Reply:
☑️ changes integrated; spelled out extract, transform, load